source/_components/alarm_control_panel.totalconnect.markdown

```diff
@@ -0,0 +1,32 @@
+---
+layout: page
+title: "Honeywell TotalConnect Alarm Control Panel"
+description: "Instructions how to integrate TotalConnect alarms into Home Assistant."
+date: 2017-04-02 22:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: honeywell-tc.png
+ha_category: Alarm
+ha_release: 0.42
+---
+
+The `totalconnect` platform provides connectivity with the Honeywell TotalConnect alarm systems used by many alarm companies
+
+To enable this, add the following lines to your `configuration.yaml`:
+
+```yaml
+# Example configuration.yaml entry
+alarm_control_panel:
+  platform: totalconnect
+  name: YOUR_PANEL_NAME #(Optional)
```

Comment:
You can leave the name out of the example configuration. Usually we try to make the example minimal, then show all the possible variables in the list below.